StereoKitC/platforms/android.cpp

```diff
@@ -495,7 +495,7 @@ font_t platform_default_font() {
 
 ///////////////////////////////////////////
 
-void platform_iterate_dir(const char* directory_path, void* callback_data, void (*on_item)(void* callback_data, const char* name, bool file)) {}
+void platform_iterate_dir(const char* directory_path, void* callback_data, void (*on_item)(void* callback_data, const char* name, bool file, const int32_t size)) {}
```

Comment:
Since we're using `int64_t` for file size, it should be consistent throughout to avoid data loss.

Reply:
Also, thinking at a higher level, is it appropriate to be passing file size through `platform_iterate_dir`? If we expand the file picker to say... include file modification and creation date, it might not make sense to be providing this data via the file discovery callback. Might be better instead to look it up in the file picker?